Stripe/STPAPIClient.m

```diff
@@ -449,3 +463,55 @@ - (void)stopPollingSourceWithId:(NSString *)identifier {
 }
 
 @end
+
+#pragma mark - Customers
+
+@implementation STPAPIClient (Customers)
+
++ (STPAPIClient *)apiClientWithEphemeralKey:(STPEphemeralKey *)key {
+    STPAPIClient *client = [[self alloc] init];
+    client.apiKey = key.secret;
```

Comment:
Ah whoops, just realized these are class methods, so we wouldn't be able to copy over instance vars. `CustomerContext` doesn't reference a configuration either, so we wouldn't be able to pass one through. I think it's fine to not set one for now.